QuowMinimap.lua

```diff
@@ -24105,8 +24108,6 @@ function OnPluginClose()
   if GetPluginInfo (sMinimapID, 17) then
     OnPluginDisable()
   end -- if enabled
-  -- Reset text rectangle
-  TextRectangle(iOldTextRectangleData[1], iOldTextRectangleData[2], iOldTextRectangleData[3], iOldTextRectangleData[4], 1, 0, 0, 0, 0)
```

Comment:
This significantly improves initial load times because it means the output pane only needs to calculate its layout once, rather than first rendering with the saved text rectangle, then again with an empty text rectangle, and then yet again with the saved text rectangle. This change improves the initial rendering phase in MUSHclient as well.However, I don't understand why the above code exists, so I want to make sure I'm not breaking anything else by removing those lines.